snap/snapcraft.yaml

```diff
@@ -8,7 +8,8 @@ description: |
   This snap contains the build environment necessary to build Mir display servers.
```

Comment:
Should this be renamed to `core22~snapcraft.yaml.in`? And then we just move it to `snapcraft.yaml` if we have 22?

Reply:
Cool :+1: